### PR TITLE
[32507] Show "Back arrow" to get from the split screen to the list again

### PR DESCRIPTION
**Pull request by HDinger**
### ToDo

- [x] Extract back button into separate component to make it reusable
- [x] Show the back button in the BIM split view only

### CODE REVIEW

File: frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts [hunk 1]
```diff
@@ -31,7 +31,6 @@ import {StateService} from '@uirouter/core';
 import {Component, Injector, OnInit} from '@angular/core';
 import {WorkPackageViewSelectionService} from 'core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-selection.service';
 import {WorkPackageSingleViewBase} from "core-app/modules/work_packages/routing/wp-view-base/work-package-single-view.base";
-import {BackRoutingService} from "core-app/modules/common/back-routing/back-routing.service";
 import {of} from "rxjs";
```
Comment (openprojectci) [inline frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts:35]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/acbe76e799f9b2a6c1841fd9da9841a6143c2da2/frontend/src/app/modules/work_packages/routing/wp-full-view/wp-full-view.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>

**Comment by HDinger** [comment]
Ticket:
https://community.openproject.com/projects/openproject/work_packages/32507/activity

**Review by oliverguenther — COMMENTED** [review]
Works fine. I noticed that when using the arrows to go to the full view, then back with the back button, and on the details page, using the back button again, will result in going to the full view.

From a technical perspective this makes sense, but we probably want to back button to always go back to the split page ?

**Comment by HDinger** [comment]
Mhh.. yes we probably want to get back to the list. I'll have a look.

**Comment by HDinger** [comment]
@oliverguenther I changed the behaviour for the split back button to always go back to the base state (the list). I also added a test for that.

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**